specs/TrackingPrevention.md

```diff
@@ -0,0 +1,205 @@
+Tracking Prevention
+===
+
+# Background
+The WebView2 team has been asked for an API to toggle tracking prevention and also ability to control levels
+of tracking prevention.
+
+We are proposing two API's

```

Comment:
It wasn't until I got to the end of the spec that I realized that the two properties are on different objects! May want to call this out more clearly.

* CoreWebView2EnvironmentOptions.IsTrackingPreventionEnabled.
* CoreWebView2Profile.TrackingProtectionLevel.

Reply:
Please make more explicit here. Start with Level property to explain feature, then define the enabled property after that in terms of the level property plus the perf benefits.